cluster-autoscaler/FAQ.md

```diff
@@ -621,7 +639,8 @@ then this node group may be excluded from future scale-ups.
 By default, scale-up is considered up to 10 seconds after pod is marked as unschedulable, and scale-down 10 minutes after a node becomes unneeded.
```

Comment:
Can we avoid duplicating content? I'd drop most of this paragraph and leave just:

> By default, scale-up is considered up to 10 seconds after pod is marked as unschedulable, and scale-down 10 minutes after a node becomes unneeded. Read [this section](...) to see how you can modify this behavior.